spec/acceptance/clock_spec.rb

```diff
@@ -20,5 +20,33 @@
     it "retains a record of the work that the job would have done" do
       expect(Clockwork::Test.block_for(test_job_name).call).to eq "Here's a running job"
     end
+
+    it "uses a custom matcher" do
+      expect(Clockwork::Test).to have_run(test_job_name)
+    end
+  end
+
+  describe "Custom matchers" do
+    subject(:clockwork) { Clockwork::Test }
+
+    before { Clockwork::Test.run(file: clock_file, max_ticks: 1) }
+
+    it { should have_run(test_job_name) }
+    it { should have_run(test_job_name).once }
+
+    it { should_not have_run(test_job_name).exactly(2).times }
+
+    it "works as an explicit expectation as well" do
+      expect(Clockwork::Test).to have_run(test_job_name)
+      expect(Clockwork::Test).to_not have_run(test_job_name).exactly(2).times
+    end
+
+    context "running multiple times" do
+      before { Clockwork::Test.run(file: clock_file, max_ticks: 2, tick_speed: 1.minute) }
+
+      it { should have_run(test_job_name).exactly(2).times }
+      it { should have_run(test_job_name) }
+      it { should have_run(test_job_name).times(2) }
```

Comment:
I think it'd also then be easy to expand the options to include other parameters like a start time and/or end time.